### PR TITLE
ARMHF publishing script - check if the Docker image is already present.

### PR DESCRIPTION
**Pull request by rdimitrov**
Proceed with building the image only if it's not already available.

Signed-off-by: Radoslav Dimitrov <rdimitrow@gmail.com>

## Motivation and Context

- [ ] I have raised an issue to propose this change ([required](https://github.com/openfaas/faas/blob/master/CONTRIBUTING.md))

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I've read the [CONTRIBUTION](https://github.com/openfaas/faas/blob/master/CONTRIBUTING.md) guide
- [ ] I have signed-off my commits with `git commit -s`
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by rdimitrov** [comment]
Oh, you're right, it's not needed.

**Comment by rdimitrov** [comment]
Applied your suggestion :+1:

File: contrib/publish-armhf.sh [hunk 1]
```diff
@@ -11,17 +11,35 @@ HERE=`pwd`
...
+   echo -e "\nBuilding: $i\n"
    git clone https://github.com/$i ./staging/$i
    cd ./staging/$i
    pwd
    export TAG=$(git describe --abbrev=0 --tags)
    echo "Latest release: $TAG"
 
-   make ci-armhf-build ci-armhf-push
+   REPOSITORY=$(get_repo_name $i)
+   TAG_PRESENT=$(curl -s "https://hub.docker.com/v2/repositories/${REPOSITORY}/tags/${TAG}-armhf/" | jq -r ".detail")
```
Comment (alexellis) [inline contrib/publish-armhf.sh]: Can we do this without a dependency on `jq`?

Comment (rdimitrov) [reply contrib/publish-armhf.sh]: Sure, I'll try to come up with an approach without any non-native dependencies :+1:

**Comment by alexellis** [comment]
I'm going to merge but I'd still live above done @rdimitrov .. 

Could you also add the operator to the list of dependencies to build?

**Comment by alexellis** [comment]
This also doesn't fail very well if lacking `jq`:

```
remote: Enumerating objects: 60, done.
remote: Counting objects: 100% (60/60), done.
remote: Compressing objects: 100% (52/52), done.
remote: Total 16320 (delta 13), reused 38 (delta 7), pack-reused 16260
Receiving objects: 100% (16320/16320), 32.60 MiB | 6.35 MiB/s, done.
Resolving deltas: 100% (7233/7233), done.
Checking connectivity... done.
/home/alex/faas/contrib/staging/openfaas-incubator/openfaas-operator
Latest release: 0.8.9
./publish-armhf.sh: line 40: jq: command not found
(23) Failed writing body
Image is already present: openfaas/openfaas-operator:0.8.9-armhf

Building: openfaas-incubator/faas-idler

```

**Pull request merged**